firmware/application/external/blespam/ui_blespam.cpp

```diff
@@ -473,23 +482,38 @@ void BLESpamView::createIosPacket(bool crash = false) {
 }
 
 void BLESpamView::createFastPairPacket() {
-    uint32_t model;
-    model = fastpairModels[rand() % fastpairModels_count].value;
+    uint32_t model = fastpairModels[rand() % fastpairModels_count].value;
+    uint8_t size = 14;
+    uint8_t* packet = (uint8_t*)malloc(size);
```

Comment:
I would have done something like this instead of all the packet[i++] lines and malloc & free, but just FYI:
`uint8_t packet[] = {3, 0x03, 0x2C,...};`

Reply:
The code could be improved, but tried to keep as close to the original as i can, so it can be easily updated if the original code changes.

Reply:
> The code could be improved, but tried to keep as close to the original as i can, so it can be easily updated if the original code changes.

Very good point!